.env

```diff
@@ -1,3 +1,3 @@
-CONSOLE_IMAGE=quay.io/securesign/rhtas-console@sha256:75966d60ed709af33efd48c53b96ea7b2fcd4608f90ccc56885bf224e34b55f5
-CONSOLE_UI_IMAGE=quay.io/securesign/rhtas-console-ui@sha256:c0b0b2d76548c05efadb2425baf93609cf6c40180f170cb531fbb7689a91db31
-CONSOLE_DB_IMAGE=registry.redhat.io/rhel9/mariadb-105@sha256:050dd5a7a32395b73b8680570e967e55050b152727412fdd73a25d8816e62d53
+CONSOLE_IMAGE=ghcr.io/securesign/rhtas-console:latest
```

Comment:
Could we use a `.env.example` with defaults, and then have a separate CI/prod environment with pinned digests? Or just add a note in the README that this is a convenience, but that CI/e2e will override with explicit images? It's just that generally we care about determinism and want to avoid `:latest` where possible.